FXyz-Samples/build.gradle

```diff
@@ -34,36 +38,50 @@ mainClassName = fxyz_sampler_mainClass
 ext.mainClass = mainClassName
 applicationDefaultJvmArgs = ["-XX:+UseConcMarkSweepGC"]
 
-configurations {
-    jdk
-}
-
-sourceSets {
-    main {
-        compileClasspath += configurations.jdk
-    }
-}
-
 dependencies {
+    compile "org.openjfx:javafx-graphics:11:win"
+    compile "org.openjfx:javafx-graphics:11:linux"
+    compile "org.openjfx:javafx-graphics:11:mac"
```

Comment:
Is adding all 3 platform a requirement for the fat jar?

Reply:
Yes, this adds the native libs of all three platforms to the fat jar.
If you don't do this, when you create the jar you will have only the current platform, so the jar won't be cross-platform

Reply:
Thanks for clarifying.